src/App/App.js

```diff
@@ -29,10 +31,18 @@ class App extends Component {
   //   this.setState({ isLoginView: !this.state.isLoginView})
```

Comment:
Make sure the error message you're setting in state has been properly translated for the end user :)